src/recall-assistant/ift-service.ts

```diff
@@ -5,7 +5,7 @@ import * as _ from 'lodash';
 export const constants = {
   // Depth of trace
   DEPTH: 30,
-  // Pagination limit to be used for trace apis
+  // Pagination limit to be used for trace APIs
   PAGE_SIZE: 30,
```

Comment:
We probably need to reduce this to 20 as we have seen that for some use cases this figures causes the API to throw a 431 error as it exceeds the max size.

Reply:
I think that makes sense, I have it as 15 because I encountered some 431 errors too.